File: pset1/mario-more.c
#include <cs50.h>
#include <stdio.h>

// Generates Super Mario style pyramid

int main(void)
{
    //VARIÁVEIS
    int altura;
    int coluna;
    int linha;

    //Pedir input
    do
    {
        altura = get_int("Qual a altura da pirâmide? ");
    }
    while (altura < 1 || altura > 8);

    for (coluna = 0; coluna < altura; coluna = coluna + 1)
    {
        // o que esta na linha
        for (linha = altura - 1; linha > coluna; linha = linha - 1)
        {
            printf(" ");
        }
        for (linha = - 1; linha < coluna; linha = linha + 1)
        {
            printf("#");
        }
            
        for (linha = 0; linha < 2; linha = linha + 1)
        {
            printf(" ");
        }
                
        for (linha = - 1; linha < coluna; linha = linha + 1)
        {
            printf("#");
        }
                    
        printf("\n");
        // fim de cada linha
    }
}

File: pset6/mario-more.py
# Implement a program that generates Super Mario style pyramid

altura = 0 
while ((altura > 8) or (altura < 1)):
    # Input do usuário 
    altura = int(input("Height: "))
    

for linha in range(altura):
    # Início da linha
    print((altura - linha - 1) * " ", end="")
    print((linha + 1) * "#", end="  ")
    print((linha + 1) * "#")
    # Fim da linha
